python/lsst/ap/verify/testPipeline.py

```diff
@@ -39,7 +39,8 @@
 from lsst.ip.diffim import GetTemplateConfig, AlardLuptonSubtractConfig, DetectAndMeasureConfig
 from lsst.pipe.tasks.characterizeImage import CharacterizeImageConfig
 from lsst.pipe.tasks.calibrate import CalibrateConfig
-from lsst.ap.association import TransformDiaSourceCatalogConfig, DiaPipelineConfig
+from lsst.ap.association import (TransformDiaSourceCatalogConfig,
+                                 DiaPipelineConfig, FilterDiaSourceCatalogConfig)
```

Comment:
This import wasn't alphabetical before, but please move it up after the `lsst.afw.table` imports

Reply:
Done!